packages/fx-core/templates/plugins/resource/aad/auth/tab/js/sso/InitTeamsFx.jsx

```diff
@@ -1,7 +1,11 @@
+// This file will initialize TeamsFx SDK and show `GetUserProfile` component after initialization.
+
 import { useTeamsFx } from "@microsoft/teamsfx-react";
 import { GetUserProfile } from "./GetUserProfile";
 
 export function InitTeamsFx() {
+  // For usage of useTeamsFx(), please refer to: https://github.com/OfficeDev/TeamsFx/tree/ga/packages/sdk-react#useteamsfx.
```

Comment:
Do we want to use some aka links here? Since the link contains branch in it and might likely change in future?